### PR TITLE
BZ1921321 and new feature on xt_32

### PR DESCRIPTION
**Pull request by kquinn1204**
@ahardin-rh This a PR for https://bugzilla.redhat.com/show_bug.cgi?id=1921321 and New feature associated with Telc-Docs https://issues.redhat.com/browse/TELCODOCS-182. I am awaiting approval on the description for https://bugzilla.redhat.com/show_bug.cgi?id=1921321

### CODE REVIEW

**Comment by kquinn1204** [comment]
@StephenJamesSmith Review feedback added will squash commit later for merge

**Comment by kquinn1204** [comment]
@ahardin-rh Known issue is here https://deploy-preview-29572--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-known-issues (BZ1921321. The new feature is in the scale section https://deploy-preview-29572--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-scale "The xt_u32 end-to-end test is enabled for CNF"

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -695,6 +695,12 @@ A new VRF CNI plugin that allows you to assign additional networks to a VRF is n
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF] and xref:../networking/hardware_networks/configuring-sriov-device.adoc#cnf-assigning-a-sriov-network-to-a-vrf_configuring-sriov-device[Assigning an SR-IOV network to a VRF].
 
+[id="ocp-4-7-scale-xt_32-end-to-end-test"]
+==== The xt_u32 end-to-end test is enabled for CNF
+xt_u32 is an iptables kernel module that allows packet filtering based on arbitrary content. It can look beyond headers or special protocols that are not covered by other iptables modules. Some of the Telco 5G partners make extensive use of this module in their software to do rule based packet routing.
+
+For more information, see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-for-platform-verification_cnf-master[Performing end-to-end tests for platform verification].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:702]: Replace the .html in the file path with .adoc 

I would also drop the sentence about 5G partners; we don't typically discuss partner behavior in our doc set.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -1151,6 +1157,20 @@ This script removes unauthenticated subjects from the following cluster role bin
...
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913826[*BZ#1913826*])
+
+* A reboot is sometimes required to enact SR-IOV changes on supported NICs. SR-IOV currently issues the reboot when it is ready. If this reboot coincides with changes in the Machine Config policy, the node can be left in an undetermined state.  The Machine Config Operator believes that the updated policy has been applied when it has not.
++
+[NOTE]
+====
+This race condition can also be caused by adding a node to a Machine Config Pool that has MCP and SR-IOV changes.
+====
++
+To avoid this issue, new nodes requiring MCO and SR-IOV changes should be completed sequentially. First, apply all MCO configuration and wait for the nodes to settle. Then apply the SR-IOV configuration.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1168]: Then, apply

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -1151,6 +1157,20 @@ This script removes unauthenticated subjects from the following cluster role bin
...
++
+If a new node is being added to a Machine Config Pool that includes SR-IOV, this issue can be avoided by removing the SR-IOV policy from the Machine Config Pool and then adding the new worker. Then re-apply the SR-IOV policy.
++
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1171]: Then, re-apply

**Review by codyhoag — APPROVED** [review]

**Pull request merged**